### PR TITLE
add release checklist

### PR DESCRIPTION
**Pull request by DavyCats**

### CODE REVIEW

File: releaseChecklist.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Release checklist
+- [ ] Check outstanding issues on JIRA and Github
+- [ ] Publish documentation (`updateDocs.sh`) from `develop` branch
+  - [ ] Copy docs folder to `gh-pages` branch
+  - [ ] Overwrite existing develop folder with docs folder on `gh-pages`
+  - [ ] Push changes to `gh-pages branch`
```
Comment (rhpvorderman) [inline releaseChecklist.md:6]: This happens automatically right? After `bash updateDocs.sh` should we list all those boxes?

File: releaseChecklist.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] Change current development version in `CHANGELOG.md` to stable version.
+- [ ] Run the release script `release.sh`
+  - [ ] Check all submodules are tagged
+  - [ ] Merge the develop branch into `master`
+  - [ ] Created an annotated tag with the stable version number. Include changes 
+    from changelog.md.
+  - [ ] Confirm or set stable version to be used for tagging
+  - [ ] Push tag to remote.
+  - [ ] Merge `master` branch back into `develop`.
+  - [ ] Add updated version number to develop
```
Comment (rhpvorderman) [inline releaseChecklist.md:18]: Same here for automated steps except the annotated tag.

File: releaseChecklist.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+- [ ] Publish documentation (`updateDocs.sh`) from `master` branch
+  - [ ] Copy docs folder to `gh-pages` branch
+  - [ ] Rename docs to new stable version on `gh-pages`
+  - [ ] Set latest version to new version
+  - [ ] Push changes to `gh-pages branch`
```
Comment (rhpvorderman) [inline releaseChecklist.md:23]: This is automated. So these things do not have to be checked off.

**Review by rhpvorderman — APPROVED** [review]

File: releaseChecklist.md (hunk 1 above)
Comment (DavyCats) [reply releaseChecklist.md:6]: I figured it might be good to add in case you might have to do it manually for one reason or another.

Comment (rhpvorderman) [reply releaseChecklist.md:6]: That's a good point.

**Pull request merged**